sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java

```diff
@@ -863,6 +866,24 @@ synchronized void getLatestReleaseDetails(String distributionGroupId, String upd
         } else {
             url += String.format(GET_LATEST_PRIVATE_RELEASE_PATH_FORMAT, mAppSecret, releaseHash);
         }
+
+        /* Send install id once upon in-app update if feature flag is enabled */
+        String shouldUseInstallIdFeature = mContext.getString(R.string.install_id_feature_enabled);
+        if(shouldUseInstallIdFeature != null && shouldUseInstallIdFeature.equals("True")) {
```

Comment:
`True` is a C# convention, not a Java one (`true`).
After changing to `true` you can simplify the check with a single statement which is the preferred way of checking boolean strings in java: `Boolean.parseBoolean(string)`.

Reply:
Thanks, great tip! Fixed.